src/submission/models.py

```diff
@@ -1287,6 +1287,15 @@ def citation_count(self):
 
         return article_link_count + book_link_count
 
+    def hidden_completed_reviews(self):
+        return self.reviewassignment_set.filter(
+            is_complete=True,
+            date_complete__isnull=False,
```

Comment:
we might as well warn them of incomplete reviews, no?

Reply:
When combined with the new decision helper (which displays incomplete reviews) I think this will be fine.